### PR TITLE
[Workflow] add guidelines for merging a PR

### PR DESCRIPTION
**Pull request by Anonymitaet**
This PR adds guidelines for merging a PR and improves visibility for doc

Preview looks good:
image

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@codelipenghui @BewareMyPower @Huanli-Meng can you help review? Thanks

File: site2/website/contributing.md [hunk 1]
```diff
@@ -15,11 +15,12 @@ We use a review-then-commit workflow in Pulsar for all contributions.
 
 **For all contributions:**
 
-1. **Code:** The best part ;-)
```
Comment (Anonymitaet) [inline site2/website/contributing.md:18]: For an open-source community, no one is the "best" part, every part (code, doc, evangelist, operation etc) is essential, they should be treated equally.

**Review by Huanli-Meng — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Review by hangc0276 — APPROVED** [review]

**Review by BewareMyPower — APPROVED** [review]

**Comment by BewareMyPower** [comment]
I have a question, is the path of `commit-message.png` correct?

The image's path is `site2/website/static/img/commit-message.png`, while the markdown's path is `site2/website/contributing.md`. However, the relative path in markdown file is `../img/commit-message.png`. Therefore, the image should be put under `site2/website/img`.

See the current view in Github: https://github.com/apache/pulsar/blob/master/site2/website/contributing.md#commit-committers-only

image

**Comment by Anonymitaet** [comment]
@BewareMyPower thanks for reminding me! I've fixed this issue in https://github.com/apache/pulsar/pull/13646/files#diff-b98c9981f2f0450fe3382b571d91b4f7bc38a2706699138e325b9c89188fe1ab and the local preview looks correct:
image

**Pull request merged**